planex/extract.py

```diff
@@ -98,7 +98,7 @@ def archive_root(tar):
     """
     Return the name of the top level directory of the tarball
     """
-    if tar.firstmember.type == tarfile.DIRTYPE:
+    if len(tar.members) == 1 and tar.members[0].isdir():
```

Comment:
Do we have components which have tarball of that kind? I am not sure that the entire logic would work with symlinks but if that is the case, and we can identify components which present such tarball, it is worth to test it.